Description: Manual pages seem to contain a hyphen where a minus sign was intended.
 <http://lintian.debian.org/tags/hyphen-used-as-minus-sign.html>
Author: Fabian Greffrath <fabian+debian@greffrath.com>

--- a/docs/bggen.man
+++ b/docs/bggen.man
@@ -2,8 +2,8 @@
 .SH NAME
 bggen \- generates colored backgrounds on X11 displays
 .SH SYNTAX
-\fBbggen\fP [-d display] [-g geometry] [-G rptgeom] [-r rot] 
-[-a] [-w width] [-h height] [-b bits] <color-spec> [<color-spec> ...]
+\fBbggen\fP [\-d display] [\-g geometry] [\-G rptgeom] [\-r rot]
+[\-a] [\-w width] [\-h height] [\-b bits] <color-spec> [<color-spec> ...]
 .SH DESCRIPTION
 \fBbggen\fP is a program that generates a rectangular 24-bit image, with
 a smooth vertical gradient between the listed colors.  (If you only specify one
@@ -14,29 +14,29 @@ in the image are interpolated between th
 more than 2 colors, the
 gradient passes through all the specified colors, in the order specified.
 .PP
-The '-d' option lets you specify what display to connect to.  \fBbggen\fP
+The '\-d' option lets you specify what display to connect to.  \fBbggen\fP
 only connects to the display to find out the size of the screen (and hence,
 the default size for the generated image).
 .PP
-The '-g' option lets you control the size of the image, as a standard
+The '\-g' option lets you control the size of the image, as a standard
 X geometry specification.  (ie, something like "512x348")
 .PP
-The '-G' option lets you control the tile size, as a standard
+The '\-G' option lets you control the tile size, as a standard
 X geometry specification.  (ie, something like "32x32")  
-Normally, (if you don't give a -G option), the gradient fills the entire
-image.  If you specify a -G option, the gradient will fill a portion of the
+Normally, (if you don't give a \-G option), the gradient fills the entire
+image.  If you specify a \-G option, the gradient will fill a portion of the
 image, and that portion will be 'center-tiled' to fill the rest of the image.
 .PP
-The '-r' option lets you rotate the gradient counter-clockwise by the
+The '\-r' option lets you rotate the gradient counter-clockwise by the
 specified number of degrees.
 .PP
-The '-a' option lets you force bggen to generate an image in PPM 'Ascii'
+The '\-a' option lets you force bggen to generate an image in PPM 'Ascii'
 format, rather than in PPM 'raw' format.
 .PP
-The '-w' and '-h' options also let you set the size of the image.
+The '\-w' and '\-h' options also let you set the size of the image.
 They are provided only for backward compatibility.
 .PP
-The '-b' option specifies the number of significant bits in the (output)
+The '\-b' option specifies the number of significant bits in the (output)
 color specifications.  It must range between 1 and 8, inclusive.  Use
 values less than 8 (the default) to limit color use by increasing the color
 granularity.  Since XV can do this sort of thing better, this is of dubious
@@ -58,27 +58,27 @@ to 'stdout'.  It's up to some other prog
 and display it.
 .PP
 To use \fBbggen\fP, you should pipe its output into an XV command, such as:
-"xv -root -quit -"
+"xv \-root \-quit \-"
 .SH TRY THESE
 .nf
 Light Blue to Dark Blue
-	bggen 100 100 255  0 0 100  | xv -root -quit  -
+	bggen 100 100 255  0 0 100  | xv \-root \-quit  \-
 RGB Rainbow
-	bggen red green blue | xv -root -quit -
+	bggen red green blue | xv \-root \-quit \-
 Green Cylinders
-	bggen black green black -h 128 | xv -root -quit -
+	bggen black green black \-h 128 | xv \-root \-quit \-
 Blue to Magenta
-	bggen blue magenta | xv -root -quit -
+	bggen blue magenta | xv \-root \-quit \-
 Full Rainbow
-	bggen black red yellow green blue purple black | xv -root -quit  -
+	bggen black red yellow green blue purple black | xv \-root \-quit  \-
 Repeating Rainbow
-	bggen red yellow green blue purple red -h 256 | xv -root -quit -
+	bggen red yellow green blue purple red \-h 256 | xv \-root \-quit \-
 Green Tiles
-	bggen green black  -r 30 -g 32x32 | xv -root -quit -
+	bggen green black  \-r 30 \-g 32x32 | xv \-root \-quit \-
 Red Balls
-	bggen red black -r 45 -g 32x32 | xv -root -quit -rmode 3 -
+	bggen red black \-r 45 \-g 32x32 | xv \-root \-quit \-rmode 3 \-
 Red+Yellow Diamonds
-	bggen red yellow  -r 45 -g 32x32 | xv -root -quit -rmode 3 -
+	bggen red yellow  \-r 45 \-g 32x32 | xv \-root \-quit \-rmode 3 \-
 .fi
 .SH AUTHOR
 John Bradley  -  bradley@dccs.upenn.edu
--- a/docs/xcmap.man
+++ b/docs/xcmap.man
@@ -2,7 +2,7 @@
 .SH NAME
 xcmap \- displays the default colormap on X11 displays
 .SH SYNTAX
-\fBxcmap\fP [ [-d] \fIdisplay\fP] [ [-g] \fIgeometry\fP]
+\fBxcmap\fP [ [\-d] \fIdisplay\fP] [ [\-g] \fIgeometry\fP]
 .SH DESCRIPTION
 \fBxcmap\fP is an X11 program that displays the contents of the colormap
 on 4, 6, and 8-bit PseudoColor (or GreyScale) displays.  It displays a grid 
--- a/docs/xv.man
+++ b/docs/xv.man
@@ -15,7 +15,7 @@ FITS, Sun Rasterfile, GIF, PCX, Targa/TG
 SCREEN$, PM, and optionally PNG, JPEG, JPEG 2000, JP2, TIFF, PostScript,
 PDF, G3 fax, MAG, PIC, MAKI (640x400), PI, and PIC2.
 .LP
-Aside from the usage screen (available by typing 'xv -help' at the command
+Aside from the usage screen (available by typing 'xv \-help' at the command
 line), documentation for XV is now distributed
 .I only
 as a PostScript (or PDF) file, as it has gotten enormous
